### ISSUE TITLE
export default class with ES6 and commonjs doesn't work

### ISSUE THREAD

**Issue by hinamiyagk** [Microsoft/TypeScript#6962]
TypeScript version: 1.7.5
For example:

hoge.ts
```ts
export default class Hoge {}
```
fuga.ts
```ts
import Hoge from "./hoge"
new Hoge();
```
`tsc hoge.ts fuga.ts -t es6 -m commonjs`
compile result:
hoge.js
```js
class Hoge {
}
exports.Hoge = Hoge;
```
fuga.js
```js
var hoge_1 = require("./hoge");
new hoge_1.default();
```
Compiled code doesn't work because `require("./hoge")` returns `{ Hoge: Hoge }` and `hoge_1.default` is undefined.

**Comment by hinamiyagk** [comment]
In typescript 1.8, this code works fine.

**Comment by jesseschalken** [comment]
I was just about to submit this.

I noticed functions work correctly:

```ts
export default function foo() {}
```
```js
function foo() { }
exports.default = foo;
```

But classes do not:

```ts
export default class Foo {}
```
```js
class Foo {
}
exports.Foo = Foo;
```

**Comment by yortus** [comment]
I think this is a duplicate of #6450 / #5844. It's fixed in v1.8beta.
